XenAdmin/Wizards/GenericPages/SelectVMStorageWithMultipleVirtualDisksPage.cs

```diff
@@ -229,7 +236,7 @@ public override void PopulatePage()
                     var cellVmDisk = new DataGridViewTextBoxCell
                                          {
                                              Tag = new StorageDetail
-                                                     {RequiredSpace = resourceData.RequiredDiskCapacity, SysId = sysId},
+                                                     {RequiredSpace = resourceData.RequiredDiskCapacity, SysId = sysId, ResourceData = resourceData},

```

Comment:
As the `StorageDetail` now contains the `ResourceData`, we could get rid of the `RequiredSpace` property and use `resourceData.RequiredDiskCapacity` instead